linera-sdk/wit/service-system-api.wit

```diff
@@ -41,6 +41,7 @@ interface service-system-api {
     record bytecode-id {
         contract-blob-hash: crypto-hash,
         service-blob-hash: crypto-hash,
+        vm-runtime: vm-runtime,
```

Comment:
@afck @jvff I'm worried that we expose too much detail about BytecodeID (and generally applications) to the SDK. Then, we will have to maintain them for ever.

Perhaps, this BytecodeID could be a hexadecimal string. Alternatively, we could also remove `create_application` for now.

Reply:
Maybe we could change. But in another PR.

Reply:
I agree that it's not ideal and will make adding new runtimes harder because it will always require changing the WIT interface.

I'm starting to lean more towards having a single bytecode blob with metadata including the VM runtime and pointing to the actual blobs to form the bytecode.

But yeah, it doesn't have to be done in this PR.